src/utils.ts

```diff
@@ -349,6 +349,18 @@ export class Utils {
     }
   }
 
+  static getContainerElement(el: HTMLElement): HTMLElement {

```

Comment:
might need to rename it better: getPositionContainer() or something. also can replace with while() loop rather than recursive...